docs/authentication/authentication.rst

```diff
@@ -48,6 +48,18 @@ The IDP may ask for the following settings:
 
 #. Custom attributes - Mautic requires three custom attributes in the IDP responses for the User Email, first name and last name. Username is also supported but is optional. Configure the attribute names used by the IDP in Mautic's Configuration > User/Authentication Settings under the 'Enter the names of the attributes the configured IDP uses for the following Mautic User fields' section.
 
+Azure Specific Configuration
+============================
+
+For Azure you need to make sure to put the complete XML schema with path instead of just Attribute name. So within Attributes do not put them like "emailaddress" or "surname". See below example for 3 required fields"
+Email: http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress
```

Comment:
As we automatically check all links and these ones are being flagged up as invalid, we should encase these links as inline code blocks.

```suggestion
Email: ``http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress``
```